cli.js

```diff
@@ -6,11 +6,12 @@ var path = require('path')
 var prog = path.resolve(process.argv[2])
 
 console.log('probing program', prog)
-console.log('kill -SIGUSR1', process.pid, 'for logging')
 
 var nodeArgs = [
   '-r',
   path.join(__dirname, 'include.js')
 ]
 var nodeOpts = { stdio: 'inherit' }
-spawn('node', nodeArgs.concat(prog), nodeOpts)
+var child = spawn('node', nodeArgs.concat(prog), nodeOpts)
```

Comment:
process.execPath instead of 'node' :)

Reply:
That's probably a better approach. Still, can we approach it in another PR as it's not affecting this one?